main/test/ca/uwaterloo/flix/api/lsp/VisitorSuite.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2024 Alexander Dybdahl Troelsen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ca.uwaterloo.flix.api.lsp
+
+import ca.uwaterloo.flix.language.ast.shared.{Input, SecurityContext, Source}
+import ca.uwaterloo.flix.language.ast.{SourceLocation, SourcePosition}
+import org.scalatest.funsuite.AnyFunSuite
+
+
+class VisitorSuite extends AnyFunSuite {
+  val source: Source = Source(Input.Text("test", "test", stable = true, SecurityContext.AllPermissions), Array.emptyCharArray)
+  val uri = "test"
+
+  test("Inside.01") {
+    val loc = SourceLocation(
+      isReal = false,
+      SourcePosition(source, 3, 10),
+      SourcePosition(source, 6, 2))
+    val pos = Position(5, 5)
+
+    assert(Visitor.inside(uri, pos)(loc) === true)
+  }
+
+  test("Inside.02") {
+    val loc = SourceLocation(
+      isReal = false,
+      SourcePosition(source, 5, 2),
+      SourcePosition(source, 5, 5),
+    )
+    val pos = Position(5, 4)
+
+    assert(Visitor.inside(uri, pos)(loc) === true)
```

Comment:
is there not an assertFalse? Otherwise assert(! should do)

Reply:
There is not an assertFalse, but yeah, assert and ! works